services/src/unit-test/java/uk/gov/moj/sdt/services/messaging/SdtMessageTest.java

```diff
@@ -0,0 +1,46 @@
+package uk.gov.moj.sdt.services.messaging;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+@ExtendWith(MockitoExtension.class)
+public class SdtMessageTest {
+
+    private static final String TEST_REFERENCE = "Test_Reference";
+    @Test
+    void getSdtRequestReferenceTest(){
+        SdtMessage sdtMessage = new SdtMessage();
+        sdtMessage.setSdtRequestReference(TEST_REFERENCE);
+
+        assertEquals(TEST_REFERENCE, sdtMessage.getSdtRequestReference());
+    }
+    @Test
+    void getMessageSentTimestampTest(){
+        SdtMessage sdtMessage = new SdtMessage();
+        sdtMessage.setMessageSentTimestamp(1L);
+
+        assertEquals(sdtMessage.getMessageSentTimestamp(),1L);
+    }
+
+    @Test
+    void getEnqueueLoggingIdTest(){
+        SdtMessage sdtMessage = new SdtMessage();
+        sdtMessage.setEnqueueLoggingId(1L);
+
+        assertEquals(sdtMessage.getEnqueueLoggingId(),1L);
```

Comment:
The assertEquals parameters should be around the other way.  The first value is the expected value (1L in this case) and the second is the actual value.

Reply:
done